### ISSUE TITLE
core-icon-buttons are not tab accessible or generally keyboard-friendly.

### ISSUE THREAD

**Issue by jklein24** [Polymer/core-icon-button#10]

**Comment by dplassgit** [comment]
I've found that you *can* tab to them, but you cannot activate a onClick handler via the keyboard.
